specification/resourcegraph/resource-manager/Microsoft.ResourceGraph/preview/2018-09-01-preview/graphquery.json

```diff
@@ -0,0 +1,544 @@
+          }
+        }
+      },
+      "put": {
+        "description": "Create a new graph query.",
+        "operationId": "GraphQuery_CreateOrUpdate",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/GraphQueryResourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "properties",
+            "description": "Properties that need to be specified to create a new graph query.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/GraphQueryResource"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The newly created graph query.",
+            "schema": {
+              "$ref": "#/definitions/GraphQueryResource"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/GraphQueryError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Create Graph Query": {
+            "$ref": "./examples/GraphQueryAdd.json"
+          }
+        }
+      },
+      "patch": {
+        "description": "Updates a graph query that has already been added.",
+        "operationId": "GraphQuery_Update",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/GraphQueryResourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "body",
+            "description": "Properties that need to be specified to create a new graph query.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/GraphQueryUpdateParameters"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The graph query definition updated.",
+            "schema": {
+              "$ref": "#/definitions/GraphQueryResource"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/GraphQueryError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Update Graph Query": {
+            "$ref": "./examples/GraphQueryUpdate.json"
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.ResourceGraph/operations": {
```

Comment:
This same path is defined in resourcegraph.json which is included in this same package, and since the return types are slightly different (nextLinkName is null in the other copy) it causes problems.  Can you please remove one of this?

Reply:
@chiragg4u Do you want me to go ahead and fix this or want to handle it separately since it is not really a scope of my PR?

Reply:
It needs to be fixed, without it we cannot generate SDKs.

Reply:
@jhendrixMSFT Thanks for the heads-up. We will fix resourcegraph.json in a separate PR.

Reply:
Are you going to remove the duplicate entry from graphquery.json then?

Reply:
Sorry if I wasn't clear, the build for this new tag `package-2018-09-preview` is broken and must be fixed before we can merge this (see red CI).  You can either remove the duplicate entry added in graphquery.json or fix resourcegraph.json so the definitions are identical (I don't know which fix is correct so you will have to decide).

Reply:
Just removed but seeing other issue and looking into it.